src/hardware/pcspeaker.cpp

```diff
@@ -214,51 +219,86 @@ void PCSPEAKER_SetCounter(Bitu cntr,Bitu mode) {
 		return;
 	}
 	spkr.pit_mode=mode;
+
+	// Activate the channel after queuing new speaker entries
+	spkr.chan->Enable(true);
+}
+
+// Returns the neutral voltage if the speaker's  fully faded,
+// otherwise returns the fallback if the speaker is active.
+float NeutralOr(float fallback)
```

Comment:
static function? (same comment for `NeutralLastPitOr`)

Reply:
Ahh yes; they operate on the static object. Done.

Reply:
`static` for functions (C keyword) has a different meaning than `static` for class methods (C++ keyword). (I presume it was late-hour typo ;))